cmd/base64decode.go

```diff
@@ -57,5 +55,5 @@ func base64decode(_ *cobra.Command, args []string) error {
 }
 
 func init() {
-	base64decodeCmd.Flags().StringVarP(&inputValue, "value", "v", "", "the value you want decode")
+	newBase64decodeCmd.Flags().StringVarP(&inputValue, "value", "v", "", "the value you want decode")
```

Comment:
flag should be changed for updating document.

Reply:
I have deleted this flag.
Because I think this flag is a little redundant.